swerve_pico/messaging-library/swerve-module-id.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Defined list of valid message ids and helper function to check and process messages into actions
+ */
+#ifndef _MESSAGING_IDS
+    #define _MESSAGING_IDS
+
+    #define MOVE_IN_DIRECTION 0x80
+
+    #define resultArraySize 6
+    
+    enum messageProcessResult {
+        PAYLOAD_ID_NOT_FOUND,
+        STATE_READ_RESULT_ARRAY,
+        STATE_ZERO_MOTORS
+    };
+
+    float resultArray[resultArraySize] = {0};
+    char flippedBinaryArray[sizeof(float) * resultArraySize] = {0};
+    /**
+     * ProcessPayload: reads parameter of specified length and performs actions based on the id
+     * 
+     * @param payload pointer to beginning of payload
+     * @param id id of payload
+     * @param len length of payload
+     * 
+     * @returns success/failure to process payload, next state of main state machine
+     */
+    char MessagingProcessPayload(char* payload, unsigned char id, unsigned char len){
+        switch(id){
```

Comment:
same thing as messaging-library.h - the function code should be defined in a cpp file, not an h file